Sources/TensorFlow/Core/TensorGroup.swift

```diff
@@ -32,8 +32,10 @@ public protocol TensorArrayProtocol {
 
     var _tensorHandleCount: Int32 { get }
```

Comment:
Should this be changed to `Int` now?

Reply:
I think so, but will do so in a subsequent CL. I have updated https://bugs.swift.org/browse/TF-542 to reflect this.